app_dart/lib/src/request_handlers/flush_cache.dart

```diff
@@ -0,0 +1,50 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:meta/meta.dart';
+
+import '../datastore/cocoon_config.dart';
+import '../request_handling/api_request_handler.dart';
+import '../request_handling/authentication.dart';
+import '../request_handling/body.dart';
+import '../request_handling/exceptions.dart';
+import '../service/cache_service.dart';
+
+/// Trigger a cache flush on a config key and return empty response if successful.
+///
+/// If [cacheKeyParam] is not passed, throws [BadRequestException].
+/// If the cache does not have the given key, throws [NotFoundException].
+@immutable
+class FlushCache extends ApiRequestHandler<Body> {
+  const FlushCache(
+    Config config,
+    AuthenticationProvider authenticationProvider, {
+    @required this.cache,
+  }) : super(config: config, authenticationProvider: authenticationProvider);
+
+  final CacheService cache;
+
+  static const String cacheKeyParam = 'key';
```

Comment:
what kind of key is this cacheKey? Will it refresh all values in the cache, including build/benchmark?

Reply:
Added documentation. This will not refresh all values, and it is not possible to do so (there's no way enumerate the keys in a subcache).

Requests that are behind CacheRequestHandler can already be passed the queryParam `flushCache` to flush that response cache.

Reply:
This is great so we can just flush the one we are interested in. Thanks.